mLRS/CommonTx/mlrs-tx.cpp

```diff
@@ -762,7 +778,28 @@ INITCONTROLLER_END
             connect_tmo_cnt--;
         }
 
-        leds.Tick_ms(connected());
+        DECc(tx_tick, SYSTICK_DELAY_MS(Config.frame_rate_ms));
+
+        if (tx_tick == 2) {
+            isInTimeGuard = true; // prevent extra work
+        }
+        if (tx_tick == 1) {
+            doPreTransmit = true; // trigger next cycle
+            pretransmit_tstamp_us = micros16();
+        }
+        if (tx_tick == 0) {
+//            crsf.TelemetryStart();
```

Comment:
Why are we no longer doing anything here?  Shouldn't we still be syncing the actual transmit with doSysTask?

Reply:
no, we sync on the doPretransmit tick, and do the transmit 500 us later

the tick==0 might be removed eventually and the other ticks deccrease by one. I was testing with when to better start the crsf and while thing, moved it now to below into the case TRANSMIT. Not sure if that is needed however, if not I would prefer it to be here.